subsys/bluetooth/audio/has_internal.h

```diff
@@ -44,6 +44,9 @@
 #define BT_HAS_CHANGE_ID_PRESET_AVAILABLE   0x02
 #define BT_HAS_CHANGE_ID_PRESET_UNAVAILABLE 0x03
 
+#define BT_HAS_PRESET_NAME_MIN 1
+#define BT_HAS_PRESET_NAME_MAX 40
+
```

Comment:
Why are these in the internal header file? 
The name is supplied by the application in `bt_has_preset_register_param` - Isn't it worth for the application to know these values?

Reply:
Done, I've improved the documentation as well